test/integration/scheduler/scoring/priorities_test.go

```diff
@@ -253,7 +253,6 @@ func TestNodeResourcesScoring(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.SidecarContainers, true)
```

Comment:
we will need to rewrite it to use emulated_version instead to test this logic

Reply:
I guess the emulated version is only needed when disabling the feature gate.
Updated the tests disabling the feature to use the emulated version.